x-pack/qa/oidc-op-tests/src/test/java/org/elasticsearch/xpack/security/authc/oidc/OpenIdConnectAuthIT.java

```diff
@@ -420,6 +420,7 @@ private PrepareAuthResponse getRedirectedFromFacilitator(String realmName) throw
             logger.info(" OpenIDConnect authentication response {}", responseBody);
             assertNotNull(responseBody.get("access_token"));
             assertNotNull(responseBody.get("refresh_token"));
+            assertNotNull(responseBody.get("realm"));
```

Comment:
This is no longer necessary since the `realm` field is removed from the response (due to presence of the `authentication` field).

Reply:
Ah, yes, late Sunday commits don't do dood )